Shower fudge updated 250423

castalyfan1012 opened the pull request:
Two values were updated in both `sbnd_full_chain_250328.cfg` and `sbnd_full_chain_data_250328.cfg`:

- TPC gain: Take as 49.529 e-/ADC for now, based on the most recent Pandora-derived gain value (https://github.com/SBNSoftware/sbndcode/pull/719/files)
- Shower fudge factor: **1/0.763**, updated from the shower reco E test in the MPVMPR v02 sample `/sdf/data/neutrino/castalyf/spine/prod_v02/mpvmpr/output_spine/larcv*_spine.h5`

Note: The previous PR has the shower fudge 1/0.762, but for now we take 1/0.763 based on the latest result by looping over _all_ MPVMPR samples/entries.


Links to the commits:
- https://github.com/DeepLearnPhysics/spine_prod/commit/71de6fa475f16c2860bedbb01124cb41a96b9e78
- https://github.com/DeepLearnPhysics/spine_prod/commit/c3849e0e3244f1227dc8618b936cef22313c66ae

Commit:
added shower quality attributes

Commit:
shower quality attributes added in sbnd data cfg

Commit:
updated shower fudge factor to 1/0.762

Commit:
TPC gain updated based on the most recent MC pandora

Commit:
updated shower fudge factor more accurately (1/0.763 now)

bear-is-asleep commented on config/sbnd/sbnd_full_chain_data_250328.cfg:
```diff
@@ -486,6 +486,17 @@ post:
+    start_mode: start_point
+    use_start_dir: false
+  shower_conversion_distance:
+    mode: vertex_to_points
```
```suggestion
```
This is handled in #11

bear-is-asleep commented on config/sbnd/sbnd_full_chain_250328.cfg:
```diff
@@ -533,6 +533,17 @@ post:
+    start_mode: start_point
+    use_start_dir: false
+  shower_conversion_distance:
+    mode: vertex_to_points
```
```suggestion
```
This is handled in #11

bear-is-asleep commented on config/sbnd/sbnd_full_chain_250328.cfg:
```diff
@@ -484,7 +484,7 @@ post:
       detector: sbnd
     gain:
-      gain: [57.,57.] #from mc #[52.356,52.356] #from pandora data
+      gain: [49.529, 49.529] # from MC pandora  #[57.,57.] #from mc #[52.356,52.356] #from pandora data
```
```suggestion
      gain: [49.529, 49.529] # from MC pandora 
```

bear-is-asleep commented on config/sbnd/sbnd_full_chain_250328.cfg:
```diff
@@ -501,7 +501,7 @@ post:
     run_mode: both
     scaling: 1.
-    shower_fudge: 1/0.77 # previous:  1/0.83
+    shower_fudge: 1/0.763 # previous:  1/0.77
```
```suggestion
    shower_fudge: 1/0.763
```
This also needs to be checked in new studies.

bear-is-asleep commented on config/sbnd/sbnd_full_chain_data_250328.cfg:
```diff
@@ -434,7 +434,7 @@ post:
       detector: sbnd
     gain:
-      gain: [52.356, 52.356] #from data
+      gain: [49.529, 49.529] #from MC pandora #[52.356, 52.356] #previous from data
```
```suggestion
      gain: [49.529, 49.529] #from MC pandora 
```

bear-is-asleep reviewed (commented):
Some residual comments to delete. Also this should be a draft, given we are not quite confident in the fudge factor.

castalyfan1012 committed:
Update config/sbnd/sbnd_full_chain_data_250328.cfg

Co-authored-by: Bear Carlson <bcarlson1@ufl.edu>

castalyfan1012 committed:
Update config/sbnd/sbnd_full_chain_250328.cfg

Co-authored-by: Bear Carlson <bcarlson1@ufl.edu>

castalyfan1012 committed:
Update config/sbnd/sbnd_full_chain_250328.cfg

Co-authored-by: Bear Carlson <bcarlson1@ufl.edu>

castalyfan1012 committed:
Update config/sbnd/sbnd_full_chain_250328.cfg

Co-authored-by: Bear Carlson <bcarlson1@ufl.edu>

castalyfan1012 committed:
Update config/sbnd/sbnd_full_chain_data_250328.cfg

Co-authored-by: Bear Carlson <bcarlson1@ufl.edu>

castalyfan1012 committed:
Update sbnd_full_chain_data_250328.cfg

SBND data gain updated for Pandora-based gain value (**47.55** now for data). Link to the docdb:
[SBN-doc-40834-v3](https://sbn-docdb.fnal.gov/cgi-bin/sso/RetrieveFile?docid=40834&filename=20250424_2025A_dev_chi2_and_hit.pdf&version=3)
